README.md

```diff
@@ -1,41 +1,44 @@
 #dwyl _Code of Conduct_
 _**Note: All of our content is [published as open source](https://github.com/dwyl/intellectual-property), so please fork and reuse this code of conduct if you like it - we'd be honoured if you did!**_
 
-<img align="right" src="http://social.beefeater.co.uk/assets/family-portrait/img/explanation/mr-happy.png"/>
+<img align="right" src="https://cloud.githubusercontent.com/assets/4185328/18620097/e2cf51e2-7e03-11e6-8a66-9fce1d709fc6.jpg"/>
+
 
 1. The [***Golden Rule***](https://en.wikipedia.org/wiki/Golden_Rule): treat others as you would like others to treat you.
 
-+ **Be happy.** We like to surround ourselves with great people so good vibes are mandatory (read: almost unavoidable).
+2. **Be happy.** We like to surround ourselves with great people so good vibes are mandatory (read: almost unavoidable).
+
+3. **If someone is doing something offensive, hurtful or that is damaging your (and others') productivity, ask them to stop.** They probably don't even realise they're doing so, just be nice about it.
 
-+ **If someone is doing something offensive, hurtful or that is damaging your (and others') productivity, ask them to stop.** They probably don't even realise they're doing so, just be nice about it.
+4. **If someone asks you nicely to _stop_ doing something, _stop_.** They're coming from a good place.
 
-+ **If someone asks you nicely to _stop_ doing something, _stop_.** They're coming from a good place.
+5. **Be human.** Behave like a ~~decent~~ *awesome* human being, as if someone you respect is always around! 
 
-+ **Be human.** Behave like a ~~decent~~ *awesome* human being, as if someone you respect is always around! 
+6. **Be respectful of other people's working hours.** People may be working when you're relaxing and vice versa so keep that in mind, whether you're chatting loudly in the same physical space as them or getting impatient about the resolution of an issue!
 
-+ **Be respectful of other people's working hours.** People may be working when you're relaxing and vice versa so keep that in mind, whether you're chatting loudly in the same physical space as them or getting impatient about the resolution of an issue!
+7. **Always be learning or teaching.****
```

Comment:
@iteles I don't know what the effect of `****` is here... I thought the max in markdown was _four_ ...